firebase-vscode/webpack.common.js

```diff
@@ -85,6 +85,37 @@ const extensionConfig = {
           ],
         },
       },
+      {
+        test: /frameworks\/utils\.ts$/,
+        loader: "string-replace-loader",
+        options: {
+          multiple: [
+            {
+              search: "require.resolve",
+              replace: "__non_webpack_require__.resolve",
+              strict: true
+            },
+            {
+              search: "require(path",
```

Comment:
This feels quite fragile - consider at least adding some explicit comments in`frameworks/utils.ts` warning about this string replace

Reply:
Added a comment.